integration-testing/rnode_testing/rnode.py

```diff
@@ -29,12 +38,12 @@ class InterruptedException(Exception):
 
 
 class NonZeroExitCodeError(Exception):
-    def __init__(self, command, exit_code, output):
+    def __init__(self, command: Tuple[Union[int, str], ...], exit_code: int, output: str):
```

Comment:
I think it should say:
```suggestion
    def __init__(self, command: Tuple[str, ...], exit_code: int, output: str):
```
I can help correcting places where `int` is passed.